spec/system/support_interface/users_spec.rb

```diff
@@ -0,0 +1,168 @@
+require "rails_helper"
+
+RSpec.describe "Identity Users Support", type: :system do
+  it "displays a list of users", vcr: true, download: true do
+    given_i_am_authorized_as_a_support_user
+    when_i_visit_the_identity_users_support_page
+    then_i_should_see_a_user
+  end
+
+  context "when there are more than 100 users" do
+    before do
+      user = {
+        "userId" => "37ee5357-fb84-478e-b750-bf552e5c8eed",
+        "email" => "kevin.e@example.com",
+        "firstName" => "Kevin",
+        "lastName" => "E",
+        "dateOfBirth" => "1990-01-01",
+        "trn" => nil
+      }
```

Comment:
Should we dry these? Most of these fields are repeated verbatim between all the different `user` variables in this file. It could be a hash of common attributes, `user_base`, that we `.merge(user_base)`.